lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3337,8 +3338,7 @@ LogicalResult ConvertAtenOp<AtenSliceTensorOp>::matchAndRewrite(
     return rewriter.notifyMatchFailure(op, "start must be a Scalar constant");
 
   if (start < 0)
-    return rewriter.notifyMatchFailure(op, "Currently unsupported: start < 0");
-
+    start = toPositiveDim(start, selfType.getShape()[dim]);
```

Comment:
nit: contrary to what the name would suggest, `toPositiveDim` is not guaranteed to output a positive dim. You have to verify this:

https://github.com/llvm/torch-mlir/blob/aee1fca2517b8bff3b18e3b01beaafe8d57a7dd8/lib/Conversion/TorchToStablehlo/Reduction.cpp#L238-L241